### ISSUE TITLE
Accuweather radar does not work

### ISSUE THREAD

**Issue by TiredOfWindows** [Feodor2/Mypal#271]
The weather radar does not work on the Accuweather web site

**Comment by Feodor2** [comment]
Used to work, now it not works on 360chrome too, shall check on newer firefox later.

**Comment by Feodor2** [comment]
Check, they fixed it on their end.

**Comment by TiredOfWindows** [comment]
Sorry, Accuweather radar still does not work. I am running Mypal 28.14.2 on a Gateway laptop with Win XP Pro 64 Ver. 2003 SP2

**Comment by Feodor2** [comment]
Well i don't know, it is working fine for me now, may be some trash of the site, it filled with it too much, breaks the whole, for me 85% blocked by ublock.

**Comment by TiredOfWindows** [comment]
Yes, it works ok on the latest Firefox on my win10. Computer. Sent via the Samsung GALAXY S® 5, an AT&T 4G LTE smartphone

-------- Original message --------
